webui/react/src/hooks/useModal/Workspace/useModalWorkspaceAddMember.tsx

```diff
@@ -74,17 +80,14 @@ const useModalWorkspaceAddMember = ({ addableUsersAndGroups, onClose }: Props):
   );
 
   const handleSelect = useCallback(
-    (value, option) => {
+    (value) => {
       const userOrGroup = addableUsersAndGroups.find((u) => {
-        if (isUser(u)) {
+        if (isUser(u) && value.substring(0, 2) === 'u_') {
```

Comment:
**non-blocking:** I think the `isUser` checks here are redundant now? If so I would remove them.

Reply:
wanted to double-check that u.id is matched to a user/group ID from the same type here and in the else block